packages/e2e-test-app/test/__snapshots__/ViewComponentTest.test.ts.snap

```diff
@@ -470,15 +466,16 @@ exports[`ViewTests Views can have overflow 1`] = `
   "children": [
     {
       "Background": null,
-      "BorderBrush": "#00000000",
-      "BorderThickness": "1,1,1,1",
+      "BorderBrush": "#E4000000",
+      "BorderThickness": "0.666666686534882,0.666666686534882,0.666666686534882,0.666666686534882",
```

Comment:
This is causing a bit of weirdness. It appears that Grid snaps it's BorderThickness to the logical pixel grid, whereas Border did not do this. I'll re-run the snapshots at 100% scaling to see if these go away, but even still, we should investigate (I have noticed that there are some trivial changes to layout due to differences in actual sizes for the thickness.